test-composer/composer.json

```diff
@@ -0,0 +1,23 @@
+{
+    "name": "roundcube/plugin-installer-test",
+    "require": {
+        "roundcube/carddav": "^4 || ^5"
+    },
+    "repositories": [
+        {
+            "type": "git",
+            "url": "https://github.com/roundcube/roundcubemail.git"
+        },
```

Comment:
```suggestion
```

@alecpl can you please submit `roundcube/roundcubemail` to Packagist and setup Packagist update hook on GitHub?

After https://github.com/roundcube/roundcubemail/pull/9407 all Roundcube libs will be usable as a library 🚀🚀